jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -135,6 +135,8 @@
 
         <extensionPoint name="explorer.nodeProcessor" interface="software.aws.toolkits.jetbrains.core.explorer.AwsExplorerNodeProcessor"/>
         <extensionPoint name="explorer.treeStructure" interface="software.aws.toolkits.jetbrains.core.explorer.AwsExplorerTreeStructureProvider"/>
+
+        <extensionPoint name="explorer.serviceNode" interface="software.aws.toolkits.jetbrains.core.explorer.nodes.AwsExplorerServiceNode"/>
```

Comment:
nit: for organization sake it would make more sense over the `nodeProcessor` and `treeStructure` that work on `serviceNode`s

Reply:
Corrected in new commit